### PR TITLE
Update(dinghy-enabled): Github Notifications

### PR DESCRIPTION
**Pull request by kinnonyee-armory**
Documentation on Github notifications and issue that can occur
Suggest all parties look at this change

### CODE REVIEW

File: content/en/docs/armory-admin/dinghy-enable.md [hunk 1]
```diff
@@ -479,7 +479,51 @@ For a complete listing of options check out the [Armory Halyard]({{< ref "armory
 {{< /tabs >}}
 
 ![Slack Notifications](/images/dinghy-slack-notifications.png)
+#### Github Notifications
+{{% alert title="New feature" %}}Github Notifications is a new feature in Armory 2.24.{{% /alert %}}
```
Comment (aimeeu) [inline content/en/docs/armory-admin/dinghy-enable.md]:
```suggestion
{{% alert title="New feature" %}}GitHub Notifications is a new feature in Armory 2.24.{{% /alert %}}
```

File: content/en/docs/armory-admin/dinghy-enable.md [hunk 2]
```diff
@@ -479,7 +479,51 @@ For a complete listing of options check out the [Armory Halyard]({{< ref "armory
...
 
+As a new feature in Armory's release of 2.24.x+, Dinghy can now provide more robust information to Github about executed pipeline changes.  This information appears as a comment in the PR.
```
Comment (aimeeu) [inline content/en/docs/armory-admin/dinghy-enable.md]:
```suggestion
As a new feature in Armory's release of 2.24.x+, Dinghy can now provide more robust information to GitHub about executed pipeline changes.  This information appears as a comment in the PR.  
```

File: content/en/docs/armory-admin/dinghy-enable.md [hunk 3]
```diff
@@ -479,7 +479,51 @@ For a complete listing of options check out the [Armory Halyard]({{< ref "armory
 {{< /tabs >}}
 
 ![Slack Notifications](/images/dinghy-slack-notifications.png)
+#### Github Notifications
+{{% alert title="New feature" %}}Github Notifications is a new feature in Armory 2.24.{{% /alert %}}
 
+As a new feature in Armory's release of 2.24.x+, Dinghy can now provide more robust information to Github about executed pipeline changes.  This information appears as a comment in the PR.  
+
+*Currently, as of 2.24.1 and 2.25.0, [Github notifications are not supported with custom endpoints and should be disabled](https://support.armory.io/support?id=kb_article&sysparm_article=KB0010290).*
```
Comment (aimeeu) [inline content/en/docs/armory-admin/dinghy-enable.md]:
```suggestion
*Currently, as of 2.24.1 and 2.25.0, GitHub notifications are not supported with custom endpoints and should be disabled. See the [KB article](https://support.armory.io/support?id=kb_article&sysparm_article=KB0010290) for details.*
```

File: content/en/docs/armory-admin/dinghy-enable.md [hunk 4]
```diff
@@ -479,7 +479,51 @@ For a complete listing of options check out the [Armory Halyard]({{< ref "armory
...
+```yaml
+notifiers:
+  github:
+    enabled: true
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+![Github Notifications](/images/armory-admin/dinghy-enable/dinghy-github-notifications.jpg)
```
Comment (aimeeu) [inline content/en/docs/armory-admin/dinghy-enable.md]:
```suggestion
![GitHub Notifications](/images/armory-admin/dinghy-enable/dinghy-github-notifications.jpg)
```

**Review by aimeeu — CHANGES_REQUESTED** [review]
Thanks Kinnon!

**Review by dogonthehorizon — APPROVED** [review]

**Review by aimeeu — APPROVED** [review]

**Comment by kinnonyee-armory** [comment]
I'm good with any and all of your changes Aimee :) Thanks again!

**Pull request merged**